phpBB/language/en/email/admin_activate.txt

```diff
@@ -7,7 +7,7 @@ The account owned by "{USERNAME}" has been deactivated or newly created, you sho
 Use this link to view the user's profile:
 {U_USER_DETAILS}
 
-Use this link to activate the account:
+Click this link to instantly activate the account:
```

Comment:
I would prefer:

> Click on this link to immediately activate the user account.

or

> Click on this link to activate the user's account at once.

Reply:
`You may activate the account immediately by clicking on this link:`